glm/detail/setup.hpp

```diff
@@ -726,6 +726,14 @@
 #	define GLM_RELAXED_CONSTEXPR const
 #endif
 
+#if GLM_LANG == GLM_LANG_CXX14 || GLM_LANG == GLM_LANG_CXX1Z
```

Comment:
It's would be more sustainable to use GLM_LANG >= GLM_LANG_CXX14

Reply:
Thanks for the suggestion.
Since I'm currently migrating our codebase to support version 0.9.9(-a2), I'll update this PR accordingly.